tests/inputs/test_sample_info.yaml

```diff
@@ -34,4 +34,18 @@ tests:
       lat_lon: 37.931388399999996 -122.0239394
     must_pass: false
     expected_failures: {}
+  - description: TEST(NER)
+    sample:
+      id: TEST:NER_1
+      description: It was found in soil.
+      env_broad_scale: 
+
+    # this is what is expected in the output
+    output:
+      id: TEST:NER_1
+      type: biolink:OntologyClass
+      term: soil
+      entity_id: ENVO:00001998
+      origin: envo.json
+
```

Comment:
Thoughts @cmungall , does this look like what you had envisioned? The next step would be assigning `soil` to  `env_broad_scale` .